From a97af92b8ad2f012a6f99eb62b0cc3537a7c1527 Mon Sep 17 00:00:00 2001
From: Rajashree Mandaogane <mandaor@amazon.com>
Date: Wed, 17 Nov 2021 15:05:04 -0800
Subject: [PATCH 31/36] Fix watch on machine object for etcdMachine to cluster
 mapper

While cherry-picking commits from 0.3.19 branch the watch got modified
by mistake. This commit fixes it by changing it back to watching Machine
objects.
---
 controllers/cluster_controller.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/controllers/cluster_controller.go b/controllers/cluster_controller.go
index ee6586ae4..2b846e5e1 100644
--- a/controllers/cluster_controller.go
+++ b/controllers/cluster_controller.go
@@ -87,7 +87,7 @@ func (r *ClusterReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manag
 	}
 
 	err = controller.Watch(
-		&source.Kind{Type: &clusterv1.Cluster{}},
+		&source.Kind{Type: &clusterv1.Machine{}},
 		handler.EnqueueRequestsFromMapFunc(r.etcdMachineToCluster),
 	)
 
-- 
2.30.1

